### PR TITLE
ci(node): upgrade to node 18

### PR DESCRIPTION
**Pull request by philibea**

### CODE REVIEW

**Comment by QuiiBz** [comment]
Looks like node 18 handles `Intl.DateTimeFormat` differently compared to node 16/17

**Comment by philibea** [comment]
> Looks like node 18 handles `Intl.DateTimeFormat` differently compared to node 16/17

yes it's not the first time we have this issue. We can ignore this actually

**Review by QuiiBz — APPROVED** [review]

**Comment by chambo-e** [comment]
> > Looks like node 18 handles `Intl.DateTimeFormat` differently compared to node 16/17
> 
> @QuiiBz yes it's not the first time we have this issue. We can ignore this actually

@philibea behaviour is different in node 14,16,17 and 18, we need to find a way to handle this properly

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node: [ '14', '16', '17' ]
+        node: ['16', '17', '18' ]
```
Comment (chambo-e) [inline .github/workflows/ci.yml:38]: 17 can be removed

**Review by Lawndlwd — APPROVED** [review]

**Review by chambo-e — APPROVED** [review]

**Pull request merged**